frontend/src/routes/constants.ts

```diff
@@ -1,7 +1,7 @@
 export default Object.freeze({
   // Public routes
   Login: '/login',
-  Dashboard: '/dashboard',
+  Dashboard: '/',
```

Comment:
I would suggest keeping this at Dashboard, and having work to automatically redirect an authenticated user to it. Remember, we will be having a splash page.

Reply:
Okee, I thought the splash page would redirect a logged in user, and since we don't have any auth'd page at '/' I thought it made sense for the dashboard to be here but I can revert :)